opendebates_emails/tasks.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from celery import shared_task
+
+from opendebates.models import Submission
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def send_email_task(type, idea_pk):
+    # Avoid circular import:
+    from opendebates_emails.models import EmailTemplate
+
+    logger.debug("send_email_task(%s, %s)", type, idea_pk)
+    try:
+        try:
+            template = EmailTemplate.objects.filter(type=type).order_by("?")[0]
```

Comment:
Do we need this try/except since we've asserted that the queryset exists before this call? I guess it is possible that it would get deleted before the task gets run...